app/controllers/users_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+
+  def new
+
+  end
+
+  def create
+
+    user = User.new(user_params)
```

Comment:
if user is saved then a message is sent to the profile page with their name

Reply:
Will we need to use sessions for future user stories?